### DISCUSSION TITLE
ledger live 2.32.2 automatic vs 2.29.0 manual

### DISCUSSION THREAD

**Discussion by dm8885555** [LedgerHQ/ledger-live-desktop#4092] [Q&A] [answered]
Please can you explain why we had to download 2.29.0 manually but are now able to download 2.32.2 automatically (as before) ?

Is it 'safe' to download 2.32.2 automatically as we did in the past ?  (as 'safe' as downloading anything obviously)

if i had 2.26.1 on another machine and never downloaded 2.29.0 can I now download 2.32.2 automatically

**Comment by gre** [comment, marked as answer]
We changed our signature authority on our Windows build which broke the "upgradability" feature of Electron Builder. Yes you can download a new build from official ledger.com/live (or on our official github here)
